transifex/utils.py

```diff
@@ -114,10 +114,10 @@ def merge(self, commit_message=github.GithubObject.NotSet, merge_method=None):
 
 
 class Repo:
-    # Make 9 attempts to merge the PR (the original attempt plus 8 retries), sleeping for 5 minutes between each
-    # attempt. This should result in a total 40 minutes of sleep time. Status checks should almost always
+    # Make 13 attempts to merge the PR (the original attempt plus 12 retries), sleeping for 5 minutes between each
+    # attempt. This should result in a total 60 minutes of sleep time. Status checks should almost always
```

Comment:
Just a suggestion, but 60 minutes may not be enough time for edx-platform as the tests can take a long time. It might be good, rather than bumping this number for all consumers of the script, to instead allow for a command line argument so edx-platform can set the number much higher if necessary.

Reply:
I will incraese the number so that overall time is 90 mins. I think its ok to bump the number for all consumers because if they finish early process will be terminated so process will always be taking same time for them.